extensions/ql-vscode/src/view/model-editor/HiddenMethodsRow.tsx

```diff
@@ -1,35 +1,41 @@
-import {
-  VSCodeDataGridCell,
-  VSCodeDataGridRow,
-} from "@vscode/webview-ui-toolkit/react";
 import * as React from "react";
 import { styled } from "styled-components";
 import { pluralize } from "../../common/word";
+import { DataGridCell, DataGridRow } from "../common/DataGrid";
+import { ModelEditorViewState } from "../../model-editor/shared/view-state";
 
-const HiddenMethodsCell = styled(VSCodeDataGridCell)`
```

Comment:
In a few cases I had to convert styled cells into two nested components. Styling on the cells themselves often seems to be ignored. Possibly this worked before because of shadow dom shenanigans.

Reply:
Did you try adding a `className` prop to the `DataGridCell` component? `styled-components` works by passing down a separate `className` to the child component, so it should work if it accepts a `className` and passes it down to its rendered component.

Reply:
Thanks for pointing that out. I hadn't realised that's how that worked. That's kinda magic.

Passing the class name prop through works and now I can add another layer of styling onto `DataGridCell`, so we can keep things like `HiddenMethodsCell` the way they are.